packages/server/src/db/index.js

```diff
@@ -82,19 +82,22 @@ async function getUser(id) {
             'users.email',
             'users.name',
             'users.role_id',
+            'users.tags',
+            'users.agency_id',
             'roles.name as role_name',
             'roles.rules as role_rules',
-            'users.agency_id',
             'agencies.name as agency_name',
             'agencies.abbreviation as agency_abbreviation',
             'agencies.parent as agency_parent_id_id',
             'agencies.main_agency_id as agency_main_agency_id',
             'agencies.warning_threshold as agency_warning_threshold',
             'agencies.danger_threshold as agency_danger_threshold',
-            'users.tags',
+            'tenants.id as tenant_id',
+            'tenants.display_name as tenant_name',
         )
         .leftJoin('roles', 'roles.id', 'users.role_id')
         .leftJoin('agencies', 'agencies.id', 'users.agency_id')
+        .leftJoin('tenants', 'tenants.main_agency_id', 'users.agency_id')
```

Comment:
I remember @chrisxkeith talking about back filling tenant ids, does that need to happen before this change goes out? Does work on this https://github.com/usdigitalresponse/usdr-gost/issues/72 need to be finished before rolling out dependencies on tenant id?

Reply:
The current state of things is that there already are dependencies on tenant id, and they were added in before we could get the database to match. Based on what @chrisxkeith has shown me all the null tenants have already been filled in, so this change should be alright now.